Source/Scene/Scene.js

```diff
@@ -2727,6 +2799,9 @@ define([
         var uniformState = context.uniformState;
 
         var drawingBufferPosition = SceneTransforms.transformWindowToDrawingBuffer(this, windowPosition, scratchPosition);
+        if (this.pickTranslucentDepth) {
+            renderForPickDepth(this, drawingBufferPosition);
```

Comment:
Are you sure `pickTranslucentDepth` need to be a member of `Scene`?  It is only checked here.